recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set version = "0.17.1" %}
-{% set number = "8" %}
+{% set number = "9" %}
 {% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '9.2' %}
 {% set build_ext_version = "1.0.0" %}
 {% set build_ext = "cuda" if cuda_enabled else "cpu" %}
 {% set build_string = "py{}h{}_{}_{}".format(CONDA_PY, PKG_HASH, number, build_ext) %}
 {% set proc_build_number = "1" %}
 
 package:
-  name: arrow-cpp-ext
+  name: libarrow
```

Comment:
```suggestion
  name: apache-arrow
```

Reply:
I think we should discuss this in https://github.com/conda-forge/conda-forge.github.io/issues/1073 - that would be a further naming scheme.

Reply:
Ah, never mind, this is for the whole package, sorry